heron/healthmgr/src/java/BUILD

```diff
@@ -14,6 +14,8 @@ healthmgr_deps_files = [
     "//heron/spi/src/java:utils-spi-java",
     "//heron/spi/src/java:metricsmgr-spi-java",
     "//heron/proto:proto-java",
+    "//heron/proto:proto_packing_plan_java",
```

Comment:
This should not be necessary, since `//heron/proto:proto-java` should contain all protos. If that's not the case would you please fix that target to make it so.